### ISSUE TITLE
Request: Mark all new entries as private

### ISSUE THREAD

**Issue by HerrRiebmann** [phw198/OutlookGoogleCalendarSync#267]
To synchronize a working and a private calendar, it would be very useful to mark one of them as private-only.
All appointments, which are created there, should automatically/always have **Sensitivity = olPrivate**.

I know, this property is already being synched. But sometimes you forget about it, and anyone in the company can read your private appointment.
Or, and that´s really bad: I cannot set the privacy via Android (7.0) calendar App. Connected to Exchange, it works, connected to GMail calendar, the property disappeared.

Changed it hardcoded on my local workingspace:
OutlookCalendar.cs
```
private void createCalendarEntry(Event ev, ref AppointmentItem ai) {
[...]
//ai.Sensitivity = (ev.Visibility == "private") ? OlSensitivity.olPrivate : OlSensitivity.olNormal;
ai.Sensitivity = OlSensitivity.olPrivate;
```
From my point of view, it will be enough, to overwrite this property on new calendar entries.
If it is already synchronized, it should keep its value.

Thanks for reading and possibly considering :)
And thank you for your great tool!!!

**Comment by phw198** [comment]
You're in luck, as I only just started work on this feature request - tracked on issue #2.

**Comment by phw198** [comment]
An early release of this feature is now available on issue #2 - I will close this ticket to keep things tidy.
